### PR TITLE
Make some overlooked strings translatable

### PR DESCRIPTION
**Pull request by Mellthas**
See the individual commits, I think these should be pretty straightforward.

### CODE REVIEW

**Comment by Mellthas** [comment]
What should I do about the test `test_terms_letter_case` complaining about `@(python: expression)`? Can the test be skipped for that line?

File: quodlibet/ext/query/missing.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ class MissingQuery(QueryPlugin, PluginConfigMixin):
     PLUGIN_NAME = _("Missing Query")
     PLUGIN_DESC = _("Matches songs without the given tag.")
     key = 'missing'
-    usage = "<b><tt>@(missing: artist)</tt></b>"
+    usage = "<b><tt>" + _("@(missing: artist)") + "</tt></b>"
```
Comment (declension) [inline quodlibet/ext/query/missing.py:20]: Is this translatable? `missing` is fixed (name of the query plugin) and `artist` is fixed (tag name)...

File: quodlibet/ext/query/pythonexpression.py [hunk 2]
```diff
@@ -18,16 +18,13 @@ class PythonQuery(QueryPlugin):
     PLUGIN_NAME = _("Python Query")
     PLUGIN_DESC = _("🐍Use Python expressions in queries.")
     key = 'python'
-    usage = """
-<b><tt>@(python: expression)</tt></b>
-
-The variable <tt>s</tt> (or <tt>a</tt>) is the song / album being matched.
+    usage = "<b><tt>" + _("@(python: expression)") + "</tt></b>\n\n" + _(
```
Comment (declension) [inline quodlibet/ext/query/pythonexpression.py:21]: Let's split this up into varaibles 

e.g. 
```python
query_syntax = _("@(python: expression)")
```
etc

File: quodlibet/ext/query/conditional.py [hunk 3]
```diff
@@ -15,7 +15,7 @@ class ConditionalQuery(QueryPlugin):
     PLUGIN_NAME = _("Conditional Query")
     PLUGIN_DESC = _("Chooses the query to match based on a condition query.")
     key = 'if'
-    usage = "<b><tt>@(if: condition-query, then-query, else-query)</tt></b>"
+    usage = "<b><tt>" + _("@(if: condition-query, then-query, else-query)") + "</tt></b>"
```
Comment (declension) [inline quodlibet/ext/query/conditional.py:18]: Hmm seeing the changes below, might be worth a helper? (I should have done this originally...)
```python
def markup_for_syntax(text: str) -> str
    return f"<b><tt>{util.escape(text)}</tt></b>"
```

File: quodlibet/ext/query/missing.py (hunk 1 above)
Comment (Mellthas) [reply quodlibet/ext/query/missing.py:20]: Oh, yes, totally missed that.

File: quodlibet/ext/query/pythonexpression.py (hunk 2 above)
Comment (Mellthas) [reply quodlibet/ext/query/pythonexpression.py]: Have done that for all the query plugins now.

File: quodlibet/ext/query/conditional.py (hunk 3 above)
Comment (Mellthas) [reply quodlibet/ext/query/conditional.py]: I have added the helper.

**Comment by Mellthas** [comment]
Added one more commit to make the usage text wrap.

**Comment by declension** [comment]
> What should I do about the test `test_terms_letter_case` complaining about `@(python: expression)`? Can the test be skipped for that line? 

Not yet but i've just been updating some of that code so maybe we can add this feature then rebase this

**Comment by Mellthas** [comment]
The tests now pass.

**Comment by declension** [comment]
Thanks!

**Pull request merged**